### ISSUE TITLE
Inconsistent documentation of tell/seek on textiobase/textiowrapper

### ISSUE THREAD

**Issue by 790aa138-7900-44f1-b5f8-825430f25423** [python/cpython#82969]
BPO | [38788](https://bugs.python.org/issue38788)
--- | :---
Nosy | @terryjreedy, @qha

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2019-11-13.14:36:28.180>
labels = ['3.8', 'library']
title = 'Inconsistent documentation of tell/seek on textiobase/textiowrapper'
updated_at = <Date 2019-11-15.20:28:43.659>
user = 'https://github.com/qha'
```

bugs.python.org fields:
```python
activity = <Date 2019-11-15.20:28:43.659>
actor = 'terry.reedy'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)']
creation = <Date 2019-11-13.14:36:28.180>
creator = 'qha'
dependencies = []
files = []
hgrepos = []
issue_num = 38788
keywords = []
message_count = 1.0
messages = ['356534']
nosy_count = 2.0
nosy_names = ['terry.reedy', 'qha']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue38788'
versions = ['Python 3.8']
```

</p></details>

**Comment by serhiy-storchaka** [comment]
Closed as a duplicate of newer issue #107801 which has more activity.

**Comment by erlend-aasland** [comment]
> Closed as a duplicate of newer issue #107801 which has more activity.

+1
